manifests/03-deployment.yaml

```diff
@@ -21,6 +21,32 @@ spec:
         controller-tools.k8s.io: "1.0"
     spec:
       containers:
+      - args:
+        - --secure-listen-address=0.0.0.0:8443
+        - --upstream=http://127.0.0.1:2112/
+        - --tls-cert-file=/etc/tls/private/tls.crt
+        - --tls-private-key-file=/etc/tls/private/tls.key
+        - --config-file=/etc/kube-rbac-proxy/config-file.yaml
+        - --logtostderr=true
+        - -v=10
+        image: quay.io/openshift/origin-kube-rbac-proxy:4.2.0
```

Comment:
One of my overall questions is where this approach came from, and here specifically is it ok to use 4.2 version hard coded? We should comment with why that's the case. 

Or is this one that is automatically replaced?

Reply:
Again, using cluster-machine-approver as an example: https://github.com/openshift/cluster-machine-approver/blob/master/manifests/04-deployment.yaml#L34

FWIW, I think I looked at a running cluster post-install and it had this same unmodified image (I'd have to double check to make sure).

Reply:
There is a kube-rbac-proxy image in the release payload. I think @sjenning  did something similar with the AWS work for pod identity where he was able to have the image replaced on the fly. Could you check if this is something we could do here to get the latest image.

Reply:
It appears that the image is being replaced even with this :4.2.0 tag. Here is the running machine-approver Pod snippet:
```
  spec:
    containers:
    - args:
      - --secure-listen-address=0.0.0.0:9192
      - --upstream=http://127.0.0.1:9191/
      - --tls-cert-file=/etc/tls/private/tls.crt
      - --tls-private-key-file=/etc/tls/private/tls.key
      - --config-file=/etc/kube-rbac-proxy/config-file.yaml
      - --logtostderr=true
      - --v=10
      image: quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:f7c7c750e954daef37c1271c2ad4ea5e2506638a7465cb251ee5b958ce290658
      imagePullPolicy: IfNotPresent
      name: kube-rbac-proxy
```
But it does seems more correct to use the :latest tag as found in https://github.com/openshift/cluster-ingress-operator/blob/master/manifests/02-deployment.yaml#L68 and https://github.com/openshift/cluster-dns-operator/blob/master/manifests/0000_70_dns-operator_02-deployment.yaml#L42 . Updated the PR to use :latest